rioxarray/xarray_plugin.py

```diff
@@ -53,6 +51,8 @@ def open_dataset(
     ):
         if open_kwargs is None:
             open_kwargs = {}
+        chunks = None
```

Comment:
Should the kwargs be removed below instead?

Reply:
yes, we can do that